src/collectd.conf.pod

```diff
@@ -8300,6 +8300,11 @@ Synopsis:
      CollectVCL         false
      CollectVSM         false
      CollectWorkers     false
+     CollectLCK         false
+     CollectMEMPOOL     false
+     CollectMGT         false
```

Comment:
As above: Unless this is an acronym, please expand to "Management" (or whatever it stands for).